src/data/apiClient.js

```diff
@@ -1,5 +1,7 @@
 import { getAuthenticatedAPIClient } from '@edx/frontend-auth';
 
+import NewRelicService from './services/NewRelicService';
+
 const apiClient = getAuthenticatedAPIClient({
```

Comment:
Now that https://github.com/edx/frontend-auth/pull/29 has merged, we should version bump `@edx/frontend-auth` and update our API client config to include the newly added callback to prevent the 403 logout.

```jsx
const apiClient = getAuthenticatedAPIClient({
  // ...

  /* I think this will be a noop function, as we don't want to actually handle the error 
   * here; rather, the component(s) making the API requests themselves should handle the error.
   */
  handleRefreshAccessTokenFailure={() => {}}
});
```